llvm/lib/Passes/PassRegistry.def

```diff
@@ -257,25 +257,29 @@ MODULE_PASS_WITH_PARAMS(
     parseStructuralHashPrinterPassOptions, "detailed;call-target-ignored")
 
 MODULE_PASS_WITH_PARAMS(
-    "default", "", [&](OptimizationLevel L) {
+    "default", "",
+    [&](OptimizationLevel L) {
```

Comment:
Same throughout this file, remove formatting diff unrelated to this change.
You can send a separate NFC PR with such changes.